src/types/__snapshots__/resolveStatements.spec.ts.snap

```diff
@@ -100,7 +100,7 @@ exports[`resolveStatements should fail statements for contract-getter-with-metho
 `;
 
 exports[`resolveStatements should fail statements for contract-initof-wrong-arg 1`] = `
-"<unknown>:26:25: Invalid type "String" for argument "owner"
+"<unknown>:26:25: Invalid type "String" for argument "owner", expected "Int"
```

Comment:
Let's rephrase the entire message like this:
```
Cannot pass value of type "String" to parameter "owner" with type "Int"
```

Reply:
have a look please